File: GNodeBGen/src/main/java/com/ericsson/tcs/topology/wrapper/CellWrapper.java
package com.ericsson.tcs.topology.wrapper;

import com.google.gson.annotations.SerializedName;

public class CellWrapper {

	@SerializedName("_CellWrapper__cellLocalId")
	private String __cellLocalId;
	
	@SerializedName("_CellWrapper__nCI")
	private String __nCI;

	/**
	 * @return the __cellLocalId
	 */
	public String get__cellLocalId() {
		return __cellLocalId;
	}

	/**
	 * @param __cellLocalId
	 *            the __cellLocalId to set
	 */
	public void set__cellLocalId(String __cellLocalId) {
		this.__cellLocalId = __cellLocalId;
	}

	/**
	 * @return the __nCI
	 */
	public String get__nCI() {
		return __nCI;
	}

	/**
	 * @param __nCI
	 *            the __nCI to set
	 */
	public void set__nCI(String __nCI) {
		this.__nCI = __nCI;
	}

}

File: GNodeBGen/src/main/resources/bin/cellWrapper.py
#!/usr/bin/python


class CellWrapper:
    
    __cellLocalId = None
    __nCI = None

    def get_cell_local_id(self):
        return self.__cellLocalId

    def get_n_ci(self):
        return self.__nCI

    def set_cell_local_id(self, value):
        self.__cellLocalId = value

    def set_n_ci(self, value):
        self.__nCI = value

